### ISSUE TITLE
Version 2.4 process unresponsive for Mojave 10.14.2 on application start

### ISSUE THREAD

**Issue by zlababa** [milgra/macmediakeyforwarder#73]
System information:
OS: Mojave 10.14.2
Hardware: MacBook Pro 15" 2017
RAM: 16 GB

[MacMediaKeyForwarder_Spindump.txt](https://github.com/milgra/macmediakeyforwarder/files/2836395/MacMediaKeyForwarder_Spindump.txt)

**Comment by emongev** [comment]
Having the exact same issue. It sometimes works, and sometimes it doesn't, although mostly it doesn't.

**Comment by milgra** [comment]
Yeah, I experienced it once, after startup but after a few minutes it started working. Trying to figure it out but I don't know how long can we keep it up against Apple's ever-changing security features :(

**Comment by endrelzr** [comment]
I have the same issue with version 2.5. Freshly installed on my new MacBook Pro, when starting the application nothing is appearing and it's not responding. Tried to kill it few times and restart but no luck. Accessibility settings were enabled, however at Automation nothing popped up. 

Mojave 10.14.3 (18D109)
MacBook Pro 15" 2018 
32GB RAM

**Comment by DaveDave2013** [comment]
Same 
System information:
OS: Mojave 10.14.2
Hardware: MacBook Pro 15" 2017
RAM: 16 GB

**Comment by acca** [comment]
Same problem for me. I hidden it from the menu bar (setting to run in background). Worked until restart then stopped. Now if I try to open the app it does not appear on the menu bar, in activity Monitoring I see the process in state `Not Responding` and looking at the logs in the console I see this error:

```
Unable to load Info.plist exceptions (eGPUOverrides)
```

**Comment by mafiacelulares** [comment]
Same problem

**Comment by victorBaro** [comment]
Same problem

**Comment by milgra** [comment]
Sorry, I have no idea what can be the problem, very rarely I run into this but most of the time it works fine.

**Comment by pedro380085** [comment]
+1

**Comment by NigelTMB** [comment]
I found the issue is can be temporarily resolve by making sure that the media apps are closed before you restart the system otherwise it wont respond until you restart again.
